components/Buttons/FavoriteButton.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-btn :absolute="absolute" icon @click.stop.prevent="toggleFavorite">
+    <v-icon medium :class="fav ? 'red--text' : ''">
+      {{ fav ? 'mdi-heart' : 'mdi-heart-outline' }}
+    </v-icon>
+  </v-btn>
+</template>
+
+<script lang="ts">
+import { BaseItemDto } from '@jellyfin/client-axios';
+import Vue from 'vue';
+
+export default Vue.extend({
+  props: {
+    item: {
+      type: Object as () => BaseItemDto,
+      required: true
+    },
+    absolute: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data() {
+    return {
+      fav: false
```

Comment:
The idea was to give instant feedback of the action and reverting it back in case it fails, for instance. 

Thanks for pointing that out, because I had in my todo list the catching of errors and I forgot about it, will add it in my follow up PR.